.gitattributes

```diff
@@ -0,0 +1,14 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+*.c text
+*.h text
+
+# Declare files that will always have CRLF line endings on checkout.
+*.sln text eol=crlf
+
+# Denote all files that are truly binary and should not be modified.
+*.png binary
+*.jpg binary
```

Comment:
Just one more thing: Ignore generated documentation files and other svg files in diffs

```suggestion
# Set the default behavior, in case people don't have core.autocrlf set.
* text=auto

# Explicitly declare text files you want to always be normalized and converted
# to native line endings on checkout.
*.c text
*.h text
*.dart text
*.sh text
*.sql text
*.md text
*.svg text -diff

# Declare files that will always have CRLF line endings on checkout.
*.sln text eol=crlf

# Denote all files that are truly binary and should not be modified.
*.png binary
*.jpg binary
*.svg binary

# Files that should be ignored for the repository's language statistics and hidden by default in diffs.
docs/database/** linguist-generated=true
docs/uml/** linguist-generated=true
```